platform/how-to/use-go-template.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: Use Go Template Functions
+sidebar_label: Use Go Template Functions
+sidebar_position: 9
+description: Learn how to use Go template functions in vCluster Platform templates.
+---
+### Overview
+Templates in vCluster Platform standardize configurations for virtual clusters, namespaces, and applications.
+You can create templates through the vCluster Platform UI or by applying manifests directly to your cluster.
+The platform provides the VirtualClusterTemplate CRD for defining templates.
+For more information, see [Creating Templates](../administer/templates/create-templates).
+
+<!-- vale off -->
+### Use Go template functions
+When defining a VirtualClusterTemplate CRD, you can use Go template functions in the same way they are used in Helm charts. 
+These functions enable conditional logic and dynamic configuration within your template values, allowing you to customize 
+deployments based on context or user input.
+
+####  Use Go template operators
+<!-- vale on -->
+```yaml title="Use eq operator to enable or disable ingresses synchronization"
+apiVersion: management.loft.sh/v1
+kind: VirtualClusterTemplate
+metadata:
+  name: vcluster-pro-template
+spec:
+  displayName: Virtual Cluster Pro Template
+  description: This virtual cluster template deploys a vCluster Pro
+  template:
+    metadata: {}
+    instanceTemplate:
+      metadata: {}
+    apps:
+      - name: helm-dashboard
+        namespace: helm-dashboard
+    pro:
+      enabled: true
+    helmRelease:
+      chart:
+        version: 0.20.0
+      values: >-
+        # Below you can configure the virtual cluster
+        controlPlane:
+          coredns:
+            embedded: true
+        sync:
+          fromHost:
+            ingressClasses:
+              enabled: true
+          toHost:
+            ingresses:
+              {{ if eq .Values.loft.project "auth-core" }}
+              enabled: true
+              {{ else }}
+              enabled: false
+              {{ end }}
+    accessPoint:
+      ingress: {}
+    spaceTemplate:
+      metadata: {}
+  access:
+    - verbs:
+        - get
+      users:
+        - '*'
+```
+#### Use template variables
+```yaml title="Use template variables to fill the configuration"
+metadata:
+  name: "{{ .Values.vclusterName }}"
```

Comment:
Not sure this is the best example considering that `.Values.loft.virtualClusterName` is provided by platform. Maybe use the platform parameter with the `cpuLimit` example to show that it is fine to mix (which should be obvious, but it doesn't hurt show it).